plugins/parodos/src/components/projectOverview/ProjectOverview.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { wrapInTestApp } from '@backstage/test-utils';
+import { render } from '@testing-library/react';
+import { useTheme } from '@material-ui/core';
+import { act } from 'react-dom/test-utils';
+
+import { ProjectOverviewPage } from './ProjectOverview';
+import { TestApp } from '../TestApp';
+
+jest.mock('@material-ui/core/styles', () => {
+  const originalModule = jest.requireActual('@material-ui/core/styles');
+  return {
+    ...originalModule,
+    useTheme: jest.fn(),
+  };
+});
+const useThemeMock = useTheme as jest.Mock;
+
+describe('ProjectOverview', () => {
+  beforeEach(() => {
+    useThemeMock.mockClear();
+  });
+
+  it('renders correctly', async () => {
+    useThemeMock.mockReturnValue({
+      transitions: {
+        duration: {
+          shortest: 100,
+        },
+      },
+    });
+
+    await act(async () => {
+      const rendered = render(
+        wrapInTestApp(() => {
+          return (
+            <TestApp>
+              <ProjectOverviewPage />
+            </TestApp>
+          );
+        }),
+      );
+      const { getByTestId, findByText, container } = rendered;
+      expect(getByTestId('header-title').textContent).toBe('Projects overview');
+      expect(getByTestId('button-add-new-project').textContent).toBe(
```

Comment:
Same as above, changing the text of a button should not break the test unless it is some hard requirement

Reply:
The point was to briefly check the rendered structure.
It can be seen as a requirement since it is part of the design. But I got your point, I can remove it.